rebar.config

```diff
@@ -10,11 +10,7 @@
 
 {minimum_otp_vsn, "23"}.
 
-{project_plugins,
- [{rebar3_hex, "~> 6.11.7"},
-  {rebar3_format, "~> 1.0.1"},
-  {rebar3_lint, "~> 1.0.1"},
-  {rebar3_edoc_extensions, "~> 0.2.4"}]}.
+{project_plugins, [{rebar3_hex, "~> 6.11.7"}, {rebar3_format, "~> 1.0.1"}]}.
```

Comment:
Please revert back those project plugins. The `rebar3_edoc_extensions` is needed for generate and publish documentation for hex.pm https://hexdocs.pm/rebar3_edoc_extensions/. This happens automatically in GitHub Actions after creating new release.

Reply:
But there is nothing documented outside of the `README.md`. Like the default empty testsuite, I feel it is worse to make people think there is documentation and when they click the link, the doc is empty.

Reply:
This is help us understand that this plugin is works as expected. This approach helped to find and fix some bugs related to relative paths of static files. This more for additional testing

Reply:
By this reason we put it here https://github.com/vkatsuba/rebar3_edoc_extensions/blob/main/.github/workflows/hex.yaml#L26. So, we can be sure that this plugin is working as expected for hex docs.